Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM 289208114389.dkr.ecr.us-east-1.amazonaws.com/picpay-dev/hyperf:php8.2-swoole5.0
```

Comment:
This seems to be a private Docker registry from AWS's ECR. Please, don't forget that OpenCodeCo's repositories are always open and public for everyone.
We should link the base image from a public registry as well.